kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -432,6 +433,7 @@ protected void writeAndFlushResponseToClient(Channel channel) {
 
                     final ByteBuf result = responseToByteBuf(response, request);
                     channel.writeAndFlush(result).addListener(future -> {
+                        requestStats.getNetworkTotalBytesOut().add(result.readableBytes());
```

Comment:
Should we check `future.isSuccess()`?

Reply:
I did not check `future.isSuccess()` because actually it is not so important.
in case of future failure we don't know if anything has been sent to the client or not.

I can add "if future.isSuccess()" if you prefer, not a big deal.

Reply:
done